.pre-commit-config.yaml

```diff
@@ -3,19 +3,19 @@ repos:
     hooks:
       - id: web-js-lint-check
         name: Web JS Lint Check
-        entry: sh ./pre-commit/web_lint_check.sh js
+        entry: python ./pre-commit/web_lint_check.py js
```

Comment:
Do you still need the `python`s if the script is executable? :)

Reply:
I have a question.
If I remove `python` command, I get `Executable `./pre-commit/web_lint_check.py` is not executable` error.
it resolves the issue by running `git update-index --chmod=+x pre-commit/web_lint_check.py`.
I don't think this command change wouldn't be reflected on repo, right?
I feel like keeping `python` command would be good.

Reply:
seems like removing `python` is not a good idea. I'll leave it alone